rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,96 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    repo:
+      # IRR REPOS ARE ADDED HERE
+      - rpc_maas:
+          irr_repo_url: https://github.com/rcbops/rpc-maas
+    series:
+      - master:
+          branch: master
```

Comment:
This should be removed because it isn't used, same for `branch:` in the template defaults.